WeatherForecast/BL/Interfaces/IWeatherService.cs

```diff
@@ -9,7 +10,7 @@ namespace BL.Interfaces
 {
     public interface IWeatherService
     {
-        Task<Weather> AddAsync(Weather user);
+        void AddAsync(Weather user);
```

Comment:
if this method is not async, then you should delete Async postfix

Reply:
was not nessesery at all, and removed.        void method